### PR TITLE
Bug 558587 implement parametrized customer export

### PR DESCRIPTION
**Pull request by eparovyshnaya**
- base fetch-export implementation for the basic export scenario - new
bundle.
 - work is in the process, but nothing is broken and the code is
mergable

Signed-off-by: elena.parovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

File: bundles/org.eclipse.passage.loc.report.core/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Automatic-Module-Name: org.eclipse.passage.loc.report.core
+Bundle-ManifestVersion: 2
+Bundle-SymbolicName: org.eclipse.passage.loc.report.core;singleton:=true
+Bundle-Version: 0.1.0.qualifier
+Bundle-Name: %Bundle-Name
+Bundle-Vendor: %Bundle-Vendor
+Bundle-Copyright: %Bundle-Copyright
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Require-Bundle: org.eclipse.passage.loc.yars.api;bundle-version="0.0.0",
+ org.eclipse.passage.lic.users
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.report.core/META-INF/MANIFEST.MF]: Please specify minimum version

File: bundles/org.eclipse.passage.loc.report.core/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -0,0 +1,13 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: org.eclipse.passage.loc.report.core
+Bundle-ManifestVersion: 2
+Bundle-SymbolicName: org.eclipse.passage.loc.report.core;singleton:=true
+Bundle-Version: 0.1.0.qualifier
+Bundle-Name: %Bundle-Name
+Bundle-Vendor: %Bundle-Vendor
+Bundle-Copyright: %Bundle-Copyright
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Require-Bundle: org.eclipse.passage.loc.yars.api;bundle-version="0.0.0",
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.report.core/META-INF/MANIFEST.MF]: 0.1.0?

File: releng/org.eclipse.passage.loc.aggregator/pom.xml [hunk 3]
```diff
@@ -71,7 +71,8 @@
 		
 		<module>../../bundles/org.eclipse.passage.loc.yars.api</module>
 	  	<module>../../tests/org.eclipse.passage.loc.yars.api.tests</module>
-	
+
+		<module>../../bundles/org.eclipse.passage.loc.report.core</module>
```
Comment (ruspl-afed) [inline releng/org.eclipse.passage.loc.aggregator/pom.xml]: we need to put before the repository and product

File: bundles/org.eclipse.passage.loc.report.core/src/org/eclipse/passage/loc/report/internal/core/Messages.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
+
+/**
+ * FIXME
+ * 
+ * @since 0.1
+ */
+public final class Messages {
```
Comment (ruspl-afed) [inline bundles/org.eclipse.passage.loc.report.core/src/org/eclipse/passage/loc/report/internal/core/Messages.java]: Please use NLS style

File: bundles/org.eclipse.passage.loc.report.core/META-INF/MANIFEST.MF (hunk 1 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.report.core/META-INF/MANIFEST.MF]: done

File: bundles/org.eclipse.passage.loc.report.core/META-INF/MANIFEST.MF (hunk 2 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.report.core/META-INF/MANIFEST.MF]: Yep

File: releng/org.eclipse.passage.loc.aggregator/pom.xml (hunk 3 above)
Comment (eparovyshnaya) [reply releng/org.eclipse.passage.loc.aggregator/pom.xml]: Done

File: bundles/org.eclipse.passage.loc.report.core/src/org/eclipse/passage/loc/report/internal/core/Messages.java (hunk 4 above)
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.loc.report.core/src/org/eclipse/passage/loc/report/internal/core/Messages.java]: it appears that NLS-style - this only string been localized - brings heavy dependency on `org.eclipse.oeg` bundle. Can we please prevent this from happening? It's is highly unwelcome change for `loc.report.core` - it's intended to be framework-independent. 

This is actually the only string for i10n. I anticipate no mess here.

**Pull request merged**